src/test/java/reactor/netty/udp/UdpServerTests.java

```diff
@@ -147,6 +148,7 @@ public void supportsUdpMulticast() throws Exception {
 					         .runOn(resources, InternetProtocolFamily.IPv4)
 					         .handle((in, out) -> {
 						         Flux.<NetworkInterface>generate(s -> {
+					                             // Suppressed "JdkObsolete" is deliberate
```

Comment:
just to clarify, maybe mention which obsolete type is deliberately used

Reply:
fixed